index.html

```diff
@@ -26,8 +26,125 @@ <h1 class="detail one">I'm Mariane<br>Glad to see<br>you</h1>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javascript</a></li>
+        </ul>
+        <a href="#" class="detail-style2">See Project</a>
+      </div>
+    </div>
+    <div class="card">
+      <img src="image/Snapshoot-Portfolio-(2).png" alt="A photo of a website page">
+      <div class="card-content">
+        <div class="card-heading">Tonic</div>
+        <ul class="tool1">
+          <li>CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>        
+        <div class="detail-style1">A daily selection of privately<br>reads:no account or<br>sign-up required</div>
+        <ul class="tool2">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javascript</a></li>
+        </ul>
+        <a href="#" class="detail-style2">See Project</a>
+      </div>
+    </div>
+    <div class="card">
+      <img src="image/Snapshoot-Portfolio-(3).png" alt="A photo of a website page">
+      <div class="card-content">
+        <div class="card-heading">Tonic</div>
+        <ul class="tool1">
+          <li>CANOPY</li>
+          <li>Back End Dev</li>
+          <li>2015</li>
+        </ul>        
+        <div class="detail-style1">A daily selection of privately<br>reads:no account or<br>sign-up required</div>
+        <ul class="tool2">
+          <li><a href="#">html</a></li>
+          <li><a href="#">css</a></li>
+          <li><a href="#">javascript</a></li>
+        </ul>
+        <a href="#" class="detail-style2">See Project</a>
+      </div>
+    </div>
+    </div>
+
+    <div class="about-section">
+      <div class="myself">
+        <h1>About<br>Myself</h1>
+        <p class="about-detail">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a href="" class="info-link">LET'S CONNECT</a>
+        <ul class="about-social">
+          <li><a href="#"><img src="image/Vector-(4).png" alt="Email icon"></a></li>
+          <li><a href="#"><img src="image/Linkedin-icon.png" alt="Social media"></a></li>
+          <li><a href="#"><img src="image/Vector-(1).png" alt="Email icon"></a></li>
+          <li><a href="#"><img src="image/Vector-(2).png" alt="Social media"></a></li>
+          <li><a href="#"><img src="image/Vector-(3).png" alt="Social media"></a></li>
+        </ul>
+        <a href="#" class="language">Get my resume</a>
+      </div>
+      <div class="language-detail">
+        <div class="lang icon">
+          <div class="word">Language</div>
+          <div class="size"><img src="image/arrow2.png" alt="dropdown arrow icon"></div>
+        </div>
+        <div class="icons">
+          <div class="space"><img src="image/Ellipse-1.png" alt="Javascript logo"></div>
+          <div class="space">Javascript</div>
+        </div>
```

Comment:
- [ ] To make things quick and simple due to the time, just change this div (126) in `li` since the main container will be changed to a `ul`. kindly apply this to all similar sections. and 
- [ ] one more thing already said above, use `hx or p` to declare your inner HTML text, please.
- [ ] Please make you stick to the design please, by saying this, you will need to use the correct `colors and font-family used in the Figma for your web app, kindly fix that too, see images below ⬇️ 
<img width="273" alt="fryuuu" src="https://user-images.githubusercontent.com/79058364/153726097-de970b30-f2f4-41a5-80d0-a3fb10040e94.png">

- [ ] ***make sure you adjust your design to fit this design below***

<img width="253" alt="fig" src="https://user-images.githubusercontent.com/79058364/153726126-9b961a69-2bee-4dfa-8191-ea932a0e82f3.png">

- [ ] Read project requirement, it seems like you have missed one of the project requirements, please consider adding this button actions too for your `see project and my resume buttton`
<img width="525" alt="btn action" src="https://user-images.githubusercontent.com/79058364/153726244-0b9d3c45-a854-442e-bc6c-59389983cbd8.png">